## Problem Statement :
Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions

## Data description
α == 0.05 (95% Confidence)

Y == Discrete

X == Discrete

Since there are more than 2 variable we will perform Chi-Square test

In [8]:
import pandas as pd 
import numpy as np 
import matplotlib.pyplot as plt 
%matplotlib inline
from sklearn.feature_selection import chi2
from scipy import stats

In [9]:
df=pd.read_csv(r"BuyerRatio.csv")
df

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

H0 =The male-female buyer rations are similar across regions

H1 =The male-female buyer rations are not similar across regions

In [10]:
import statsmodels.formula.api as smf

In [11]:
stats.chi2_contingency([df["East"],
                        df["West"],
                        df["North"], 
                        df["South"]])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value of Chi-Square test == 0.66 > α

Thus HO is accepted.
The male-female buyer rations are similar across regions